internal/builders/container/README.md

```diff
@@ -321,6 +322,84 @@ Certificate issuer URL:  https://token.actions.githubusercontent.com
 
 You can read more in the [cosign documentation](https://docs.sigstore.dev/cosign/attestation/).
 
+### Sigstore policy-controller
+
+Sigstore [policy-controller](https://docs.sigstore.dev/policy-controller/overview)
+is a policy management controller that can be used to write Kubernetes-native
+policies for SLSA provenance. The following assumes you have
+[installed policy-controller](https://docs.sigstore.dev/policy-controller/installation)
+in your Kubernetes cluster.
+
+The following `ClusterImagePolicy` can be used to verify container images
+as part of [Admission Control](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/).
+
+```yaml
+apiVersion: policy.sigstore.dev/v1alpha1
+kind: ClusterImagePolicy
+metadata:
+  name: image-is-signed-by-github-actions
+spec:
+  images:
+    # All images in example repository matched
+    - glob: "**"
```

Comment:
Maybe update this to be more specific and match the kyverno policy below.

Maybe something like: `glob: "ghcr.io/ianlewis/actions-test:*"`